tests/run.sh

```diff
@@ -54,3 +61,22 @@ for options in -O0 -O2 "-O2 -flto"; do
     done
 done
 cd - >/dev/null
+
+if [[ -n "$wasi_sdk" ]]; then
+    for option in Debug Release; do
+        rm -rf "$testdir/cmake/build/$option"
+        mkdir -p "$testdir/cmake/build/$option"
+        cd "$testdir/cmake/build/$option"
+        cmake \
```

Comment:
Maybe we just need to add a `-G "Unix Makefiles"` around here somewhere to make this work? I'm not strongly opposed to generating Ninja files but since (a) we know we have `make` working in all of the CI agents and (b) this project already used `Makefile`s extensively, maybe we should just keep going with that? Or did I misunderstand something about your comment?